cartographer_ros/cartographer_ros/node.cc

```diff
@@ -679,14 +679,16 @@ bool Node::HandleStartTrajectory(
     }
 
     // Check if the requested trajectory for the relative initial pose exists.
-    response.status = TrajectoryStateToStatus(
-        request.relative_to_trajectory_id,
-        {TrajectoryState::ACTIVE, TrajectoryState::FROZEN,
-         TrajectoryState::FINISHED} /* valid states */);
-    if (response.status.code != cartographer_ros_msgs::StatusCode::OK) {
-      LOG(ERROR) << "Can't start a trajectory with initial pose: "
-                 << response.status.message;
-      return true;
+    if (request.relative_to_trajectory_id != -1) {
```

Comment:
Perhaps add a comment? "No need to check for existence of virtual trajectory -1, which is fixed in the map origin frame"